samples/tab-user-profile/ts/src/components/profileTab.tsx

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import React, { useEffect, useState } from 'react';
+import { app, profile } from "@microsoft/teams-js";
+import validator from 'validator';
+import './Tab.css';
+
+// function for opening user profile card
```

Comment:
// Function for opening user profile information card.

Reply:
Fixed